src/cmap/wire_protocol/responses.ts

```diff
@@ -391,3 +391,9 @@ export class ClientBulkWriteCursorResponse extends CursorResponse {
     return this.get('writeConcernError', BSONType.object, false);
   }
 }
+
+export class CreateSearchIndexesResponse extends MongoDBResponse {
+  get indexesCreated() {
```

Comment:
I don't feel strongly here but I've been putting response classes in the operation classes because they're not used anywhere else (no need to export).  Do you have a preference where we put them?

Reply:
Happy to do that. No preference.

Reply:
Moved